x11-misc/xpad/xpad-5.7.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools
+
+DESCRIPTION="A sticky note application for GTK"
+HOMEPAGE="https://launchpad.net/xpad"
+SRC_URI="https://launchpad.net/${PN}/trunk/${PV}/+download/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~x86"
+
+RDEPEND="
+	app-accessibility/at-spi2-atk
+	dev-libs/atk
+	>=dev-libs/glib-2.58:2
+	sys-devel/gettext
```

Comment:
This looks weird. I'm gonna remove it from RDEPEND but please do check whether it's needed there or not.